keras_hub/src/models/retinanet/retinanet_backbone.py

```diff
@@ -0,0 +1,84 @@
+import keras
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.feature_pyramid_backbone import FeaturePyramidBackbone
+from keras_hub.src.models.retinanet.feature_pyramid import FeaturePyramid
+from keras_hub.src.utils.keras_utils import standardize_data_format
+
+
+@keras_hub_export("keras_hub.models.RetinaNetBackbone")
+class RetinaNetBackbone(FeaturePyramidBackbone):
+    def __init__(
+        self,
+        backbone,
+        min_level,
+        max_level,
+        image_shape=(None, None, 3),
+        data_format=None,
+        dtype=None,
+        **kwargs,
+    ):
+
+        if min_level > max_level:
+            raise ValueError(
+                f"Minimum level ({min_level}) must be less than or equal to "
+                f"maximum level ({max_level})."
+            )
+
+        data_format = standardize_data_format(data_format)
+        input_levels = [int(level[1]) for level in backbone.pyramid_outputs]
+        backbone_max_level = min(max(input_levels), max_level)
+        image_encoder = keras.Model(
```

Comment:
ooo, we have an image_encoder here. haha, any suggestions for name here other than image_encoder

Reply:
for input args: `image_encoder` (Backbone)
for pyramid outputs extraction: `feature_extractor` (Resnet with feature pyramid outputs)